server/src/types/resolvers/Channel/mutation.ts

```diff
@@ -228,4 +211,4 @@ export const leaveChannel = mutationField('leaveChannel', {
       },
     });
   },
-});
+});
```

Comment:
Why removing new line at the end of the code? This is more likely to have one.
Also, try not to change any styles of your code if that is not the purpose of the code.

This is in our [CONTRIBUTING coding guideline](https://github.com/dooboolab/hackatalk/blob/master/CONTRIBUTING.md#coding-guidelines)
![Screenshot 2020-08-20 at 10 47 08 PM](https://user-images.githubusercontent.com/27461460/90778066-26177780-e337-11ea-9cd5-d5e6c8176b4c.png)

Reply:
I will keep in mind.